### PR TITLE
Add formatting + prettier config

### PR DESCRIPTION
**Pull request by tobezdev**
**This is not a domain registration PR.**

This PR is my attempt at trying to help out by formatting all of the files correctly including installing a formatter that can be easily run for all future files.

Summary of changes in this PR:
* [+] `.prettierrc` - formatting for the future formatting runs;
* [~] `package-lock.json` - add packages for formatting modules;
* [~] `domains/*` - format all existing files as of commit id: **26f3ffd5857d6d682dd94ca134bfada3951f5a7f**

If you have any questions, feel free to reply to this thread or message me on socials.

### CODE REVIEW

**Review by MaskDuck — CHANGES_REQUESTED** [review]
first of all, I highly appreciate your time and effort into this PR. this PR is quite good and you have made a good attempt to make the JSON more readable and I would like to personally thank you for that (that does make our job easier!) however, I've got some concerns about this PR below (slight changes only, dw). if you think that changes don't need to be made then consider it as an approval from me :) (I should have pressed comment instead of request changes tbh)

we actually *do need* a formatter through and thank you for adding it, it will be even nicer of you if you can have a GH workflow for that! (would prefer one that runs on the main repo on a schedule)

(also if you can find a way to format JSON files only then it would be nice, I'd prefer it if we keep the markdown files as it is except for the table :P )

we've got a discord server too! (find it in the README, I think it should be somewhere)

**Comment by wdhdev** [comment]
@TobezDev Please close this PR and just add a workflow which runs once a week using the prettier config, except with only 2 tab width spaces.

File: README.md [hunk 1]
```diff
@@ -17,33 +17,44 @@
...
 <a href="https://liberapay.com/phenax" target="_blank"><img src="https://img.shields.io/badge/liberapay-donate-yellow.svg?style=for-the-badge" alt="Liberapay"></a>
 
 ### Similar Services
+
 If you want to find services similar to is-a.dev, take a look on [Free For Life](https://github.com/wdhdev/free-for-life#domains).
+
+### Formatting
+
+Formatting was added to this repo by @TobezDev.<br>
+To execute formatting, use: `npx prettier <> --write` where <> is replaced with the directory you wish to format. Use a period to format the entire current directory.
```
[the remark this answers is not in the document]
Comment (wdhdev) [reply README.md:60]: Yeah these will not be added.
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -17,33 +17,44 @@
...
+-   [Read the documentation](https://www.is-a.dev/docs).
+-   Your pull request will be reviewed and merged. _Make sure to keep an eye on it incase we need you to make any changes!_
+-   After the pull request is merged, please allow up to 24 hours for the changes to propagate.
+-   Enjoy your new `.is-a.dev` domain!
 
 ### Automated Registration
-> [!WARNING]
-> **When using automated registration, please ensure your records are valid. Please do NOT "test" out the system and create invalid PRs, it wastes maintainers' time.**
+
+> [!WARNING] > **When using automated registration, please ensure your records are valid. Please do NOT "test" out the system and create invalid PRs, it wastes maintainers' time.**
```
[the remark this answers is not in the document]
Comment (wdhdev) [reply README.md:38]: I believe so

[the remark this answers is not in the document]
Comment (TobezDev) [reply README.md:38]: I'm working on a remake for formatter configuration, please hang tight! I'm working on converting the `npx prettier . --write` into a GH workflow on a schedule. Thinking of having the workflow run on-commit and only formatting the files that have been modified/added.

**Comment by TobezDev** [comment]
Closing as requested by @wdhdev - creating a new pr to add this as a workflow instead of manual running of `npx prettier . --write`.

**Pull request closed without merging**